cfpb/sbl-frontend
# fix(deps): add latest DSR, includes DS 0.35.0 updates

A companion to [#289](https://github.com/cfpb/design-system-react/pull/289) in the DSR. 

## Changes

- Runs `yarn up -i design-system-react`

## How to test this PR

1. Do you see the updates to the DS like focus error states on checkboxes on "Complete your user profile"? (hover error state is coming in a later PR)

## Screenshots
![Screenshot 2024-01-30 at 3 58 20 AM](https://github.com/cfpb/sbl-frontend/assets/19983248/ac848f17-d796-42e6-aa8f-153049ac8e28)


## Notes

- more DSR changes are coming, but we should keep this updated
